src/mango/src/mango_cursor_view.erl

```diff
@@ -384,9 +409,15 @@ view_cb({row, Row}, #mrargs{extra = Options} = Acc) ->
             Process(Doc)
     end,
     {ok, Acc};
-view_cb(complete, Acc) ->
+view_cb(complete, #mrargs{extra = Options} = Acc) ->
+    case couch_util:get_value(execution_stats_map, Options) of
```

Comment:
It look a bit cleaner to have a default `false` value and then explicitly match on `true` or `false`. What do you think?

Reply:
we should be explicit.